apps/aefate/test/aefa_sophia_test.erl

```diff
@@ -109,15 +147,82 @@ print_run_stats(Time, ES) ->
     GasUsed    = ?CALL_GAS - aefa_engine_state:gas(ES),
     Trace      = aefa_engine_state:trace(ES),
     Red        = fun({_, {reductions, R}}) -> R end,
-    Reductions = Red(hd(Trace)) - Red(lists:last(Trace)),
+    Reductions = Red(hd(Trace ++ [{bla, {reductions, 0}}])) - Red(lists:last([{bla, {reductions, 0}} | Trace])),
```

Comment:
For what is `bla` being used?

Reply:
I don't want to crash if `Trace` is empty. As you can see in the line above, the first component of the pair is ignored, `bla` is just a dummy value.

Reply:
Ah I see, thanks.